src/Components/App.js

```diff
@@ -59,20 +60,20 @@ class App extends Component {
     const error =  this.state.error && <h1>{this.state.error}</h1>
     return (
       <>
-      <NavBar searchMovie={this.searchMovie} />
-      {error}
-      <ErrorBoundary error={this.state.error}>
-        <Route exact path="/" render={() => <Movies movieInfos={this.state.movieInfo} />} />
-      </ErrorBoundary>
+        <NavBar searchMovie={this.searchMovie} />
```

Comment:
We learned so much in our figuring out of Route and Switch, and how Switch can only have one child!